lib/config_editor/config_editor_dialog.dart

```diff
@@ -82,7 +82,34 @@ class ConfigEditor extends StatelessWidget {
           ),
         );
         break;
-      case bool:
+      case 'blob':
+      case 'multiline string':
+        final controller = ScrollController();
```

Comment:
the controller is not disposed, but is the controller actually needed?

Reply:
ok, got confirmed that it's needed so we just need to find a way to dispose it after use

Reply:
Not sure if there's a better way of doing this - I've added a stateful widget to handle this now